package.json

```diff
@@ -86,13 +86,11 @@
     "lodash.escaperegexp": "^4.1.2",
     "lodash.kebabcase": "^4.1.1",
     "lodash.startcase": "^4.4.0",
-    "postcss": "^6.0.21",
```

Comment:
This still may be needed for autoprefixer or the postcss-loader unless those pull in their own version of postcss.

Reply:
Terra-toolkit actually defines this